docs/installing_wash.md

```diff
@@ -18,5 +18,8 @@ Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here
 
 * Run `wash --verify-install` to ensure that the installation was successful
     * If anything fails, then check out the [known issues page]({{ '/known_issues' | relative_url }}) to see if the failure(s) correspond to any of the known issues. Otherwise, please don't hesitate to ask us on [slack](https://puppetcommunity.slack.com/app_redirect?channel=wash) for help!
+        * You can use `wash --verify-install` to test any fixes
```

Comment:
I don't think I'd indent this, putting at the same level as the previous note seems fine.

Reply:
Yeah, putting on the same level does look a bit better. Updated.